spec/codecs/json_lines_spec.rb

```diff
@@ -58,6 +58,48 @@
         insist { decoded } == true
       end
     end
+
+    context "when json could not be parsed" do
+
+      let(:message)    { "random_message" }
+
+      it "add the failure tag" do
+        subject.decode(LogStash::Json.dump(message)) do |event|
```

Comment:
Also no need to do `LogStash::Json.dump(message)` as this is done internally in the decode. You can just pass `message` to `subject#decode`